### ISSUE TITLE
wx.ChoiceBook.GetControlSizer() does not appear to be implemented

### ISSUE THREAD

**Issue by Randalphwa** [wxWidgets/Phoenix#2335]
**Operating system**: Windows (probably all operating systems)
**wxPython version & source**:  4.2.0 installed from pypi.org
**Python version & source**:   3.10 stock

**Description of the problem**:

The documentation for **wx.ChoiceBook** states that: "wx.Choicebook allows the use of BookCtrlBase.GetControlSizer(), allowing a program to add other controls next to the choice control.". However, using this causes an error.

Example:

```python
    page_sizer_1 = wx.BoxSizer(wx.VERTICAL)
    self.m_choicebook = wx.Choicebook(page_2, wx.ID_ANY)
    page_sizer_1.Add(self.m_choicebook, wx.SizerFlags().Border(wx.ALL))
    btn = wx.Button(self.m_choicebook, wx.ID_ANY, "First")
    self.m_choicebook.GetControlSizer().Add(btn, wx.SizerFlags().Expand().Border(wx.ALL))
```

Running the above code results in the error message:

```
AttributeError: 'Choicebook' object has no attribute 'GetControlSizer'. Did you mean: 'GetContainingSizer'?
```

And no, `GetContainingSizer()` does _not_ work as a replacement as the error message suggests.

Note that the following C++ code works fine:

```C++
    auto* page_sizer_1 = new wxBoxSizer(wxVERTICAL);
    m_choicebook = new wxChoicebook(page_2, wxID_ANY);
    page_sizer_1->Add(m_choicebook, wxSizerFlags().Border(wxALL));
    auto* btn = new wxButton(m_choicebook, wxID_ANY, "First");
    m_choicebook->GetControlSizer()->Add(btn, wxSizerFlags().Expand().Border(wxALL));
```

**Comment by Randalphwa** [comment]
I don't suppose there's any chance it _could_ be implemented in wxPython since the function is part of the wxWidgets **wxBookCtrlBase** class and is documented in the main wxWidgets docs?  See https://docs.wxwidgets.org/trunk/classwx_choicebook.html

**Comment by swt2c** [comment]
No, it certainly *can* be implemented.  I was just pointing out why it wasn't previously.  (wxPython is generated semi-automatically from wxWidgets interface headers.)
